[
   {
      "author_association" : "MEMBER",
      "body" : "See also #28115",
      "created_at" : "2023-09-25T12:23:48Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/28529#issuecomment-1733606084",
      "id" : 1733606084,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28529",
      "node_id" : "IC_kwDOABII585nVLrE",
      "performed_via_github_app" : null,
      "reactions" : {
         "+1" : 0,
         "-1" : 0,
         "confused" : 0,
         "eyes" : 0,
         "heart" : 0,
         "hooray" : 0,
         "laugh" : 0,
         "rocket" : 0,
         "total_count" : 0,
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1733606084/reactions"
      },
      "updated_at" : "2023-09-25T12:24:08Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1733606084",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/6399679?v=4",
         "events_url" : "https://api.github.com/users/MarcoFalke/events{/privacy}",
         "followers_url" : "https://api.github.com/users/MarcoFalke/followers",
         "following_url" : "https://api.github.com/users/MarcoFalke/following{/other_user}",
         "gists_url" : "https://api.github.com/users/MarcoFalke/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/MarcoFalke",
         "id" : 6399679,
         "login" : "MarcoFalke",
         "node_id" : "MDQ6VXNlcjYzOTk2Nzk=",
         "organizations_url" : "https://api.github.com/users/MarcoFalke/orgs",
         "received_events_url" : "https://api.github.com/users/MarcoFalke/received_events",
         "repos_url" : "https://api.github.com/users/MarcoFalke/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/MarcoFalke/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/MarcoFalke/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/MarcoFalke"
      }
   },
   {
      "author_association" : "MEMBER",
      "body" : "> ```\r\n> PermissionError: [WinError 32] The process cannot access the file because it is being used by another process...\r\n> ```\r\n\r\nTrying to collect some details with Sysinternals `handle64.exe` makes the issue disappear :man_shrugging: ",
      "created_at" : "2023-09-25T21:38:21Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/28529#issuecomment-1734499609",
      "id" : 1734499609,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28529",
      "node_id" : "IC_kwDOABII585nYl0Z",
      "performed_via_github_app" : null,
      "reactions" : {
         "+1" : 0,
         "-1" : 0,
         "confused" : 0,
         "eyes" : 0,
         "heart" : 0,
         "hooray" : 0,
         "laugh" : 0,
         "rocket" : 0,
         "total_count" : 0,
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1734499609/reactions"
      },
      "updated_at" : "2023-09-25T21:38:21Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1734499609",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/32963518?v=4",
         "events_url" : "https://api.github.com/users/hebasto/events{/privacy}",
         "followers_url" : "https://api.github.com/users/hebasto/followers",
         "following_url" : "https://api.github.com/users/hebasto/following{/other_user}",
         "gists_url" : "https://api.github.com/users/hebasto/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/hebasto",
         "id" : 32963518,
         "login" : "hebasto",
         "node_id" : "MDQ6VXNlcjMyOTYzNTE4",
         "organizations_url" : "https://api.github.com/users/hebasto/orgs",
         "received_events_url" : "https://api.github.com/users/hebasto/received_events",
         "repos_url" : "https://api.github.com/users/hebasto/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/hebasto/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/hebasto/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/hebasto"
      }
   },
   {
      "author_association" : "MEMBER",
      "body" : "https://github.com/bitcoin/bitcoin/actions/runs/7433919780/job/20227517742:\r\n```\r\n2024-01-06T21:17:20.926000Z TestFramework (INFO): Tests successful\r\n\r\n\r\n\r\nstderr:\r\nTraceback (most recent call last):\r\n\r\n  File \"D:\\a\\bitcoin\\bitcoin/test/functional/tool_utxo_to_sqlite.py\", line 111, in <module>\r\n\r\n    UtxoToSqliteTest().main()\r\n\r\n  File \"D:\\a\\bitcoin\\bitcoin\\test\\functional\\test_framework\\test_framework.py\", line 154, in main\r\n\r\n    exit_code = self.shutdown()\r\n\r\n                ^^^^^^^^^^^^^^^\r\n\r\n  File \"D:\\a\\bitcoin\\bitcoin\\test\\functional\\test_framework\\test_framework.py\", line 363, in shutdown\r\n\r\n    shutil.rmtree(self.options.tmpdir)\r\n\r\n  File \"C:\\hostedtoolcache\\windows\\Python\\3.12.1\\x64\\Lib\\shutil.py\", line 808, in rmtree\r\n\r\n    return _rmtree_unsafe(path, onexc)\r\n\r\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\r\n\r\n  File \"C:\\hostedtoolcache\\windows\\Python\\3.12.1\\x64\\Lib\\shutil.py\", line 636, in _rmtree_unsafe\r\n\r\n    onexc(os.unlink, fullname, err)\r\n\r\n  File \"C:\\hostedtoolcache\\windows\\Python\\3.12.1\\x64\\Lib\\shutil.py\", line 634, in _rmtree_unsafe\r\n\r\n    os.unlink(fullname)\r\n\r\nPermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'D:\\\\a\\\\_temp\\\\test_runner_â¿_ð_20240106_210717\\\\tool_utxo_to_sqlite_113\\\\utxos.sqlite'\r\n```",
      "created_at" : "2024-01-08T14:29:13Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/28529#issuecomment-1881119075",
      "id" : 1881119075,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28529",
      "node_id" : "IC_kwDOABII585wH5lj",
      "performed_via_github_app" : null,
      "reactions" : {
         "+1" : 0,
         "-1" : 0,
         "confused" : 0,
         "eyes" : 0,
         "heart" : 0,
         "hooray" : 0,
         "laugh" : 0,
         "rocket" : 0,
         "total_count" : 0,
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1881119075/reactions"
      },
      "updated_at" : "2024-01-08T14:29:13Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1881119075",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/32963518?v=4",
         "events_url" : "https://api.github.com/users/hebasto/events{/privacy}",
         "followers_url" : "https://api.github.com/users/hebasto/followers",
         "following_url" : "https://api.github.com/users/hebasto/following{/other_user}",
         "gists_url" : "https://api.github.com/users/hebasto/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/hebasto",
         "id" : 32963518,
         "login" : "hebasto",
         "node_id" : "MDQ6VXNlcjMyOTYzNTE4",
         "organizations_url" : "https://api.github.com/users/hebasto/orgs",
         "received_events_url" : "https://api.github.com/users/hebasto/received_events",
         "repos_url" : "https://api.github.com/users/hebasto/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/hebasto/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/hebasto/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/hebasto"
      }
   }
]
